public/404/404.php

```diff
@@ -0,0 +1,28 @@
+<?php require_once("../../private/initialize.php"); ?>
+
+<?php
+    
+    $page_title = "Yabe | Browse | Products";
+    $style_sheets = [
+        "/css/common.css",
+        "/css/404.css",
+    ];
+    $scripts = [
+        "/js/common.js",
+    ];
+    
+    include(SHARED_PATH . "/top.php");
+
+?>
+    <body>
+    <div class="container">
+        <div class="first-4">4</div>
+        <img class="far" src="../media/image/yabe_cursor.gif" alt="YABE crying face">
+        <div class="second-4">4</div>
+        <div class="message">SORRY PAGE NOT FOUND
```

Comment:
`SORRY PAGE NOT FOUND` should be in its own `<h2>` tag to increase its contrast from the "Don't cry message" bellow. Right now both has the same font-size and the same font-weight. This is a classic typography design mistake. Remember the [Oscars 2017 Academy Award for Best Picture incident](https://www.good.is/articles/oscars-mistake-typography)? It is that same mistake.

Wording: Can you add a comma after "SORRY" and a period after "FOUND": SORRY, PAGE NOT FOUND.